### ISSUE TITLE
TBC support

### ISSUE THREAD

**Issue by AcidWeb** [AcidWeb/CurseBreaker#290]
**EDIT** Please read: https://github.com/AcidWeb/CurseBreaker/issues/290#issuecomment-842166783

Currently TBC support is more or less broken.
There is some confusion about future of Classic client - It will still exist? Both TBC and Classic realms will use the same client?
Until situation clarify I don't plan to modify CB code as addon sites also don't know fully how handle it.

**Comment by AcidWeb** [comment]
This will be a bumpy ride. Classic client will be updated to BC - and still be called classic. "Old" classic will be released as new separate version of client: classic_era.

**Comment by AcidWeb** [comment]
I plan to release BC support same day as pre-patch launch. After I check the state of CurseForge.

**Comment by Glaived** [comment]
How will CB manage the update on the TBC side? We will certainly have to update CB, but what about the addons (because they were installed in the classic compatible version and not TBC)?

**Comment by AcidWeb** [comment]
If CurseForge API will not fail (and I have some doubts about it) they will be updated to TBC compatible versions.
WoWI and TukUI (ElvUI, Tukui excluded) ones most probably will require re installation to switch to new TBC compatible project.
GitHub ones will switch to BC releases automatically.

TLDR: Expect issues.

**Comment by AcidWeb** [comment]
CF will not fix any of underlying issues before BC prepatch. If any author will release package that is marked to support classic and bc at the same time it will be delivered only to the one client type.

Same issue as retail/classic packages.

**Comment by AcidWeb** [comment]
v3.18 is ready and it will be released tomorrow.

Breakage of update process is expected. Due to nature of changes all addon installed from CurseForge will attempt to switch from Classic flavor to BC one - and fail if BC compatible package will not be released yet. So expect errors.
Addons installed from GitHub should switch to BC packages automatically.
As other sources store BC releases as separate projects so they will need to be uninstalled and installed anew.

Retail is unaffected.

Classic Era client will be supported too. CB config from new Classic client can be moved of Classc Era one.
